.travis.yml

```diff
@@ -7,6 +7,6 @@ os:
 script:
 - curl -sSL https://raw.github.com/ndmitchell/hlint/master/misc/travis.sh | sh -s .
 - curl -sSL https://get.haskellstack.org/ | sh
-- stack setup > nul
+- stack setup > /dev/null
```

Comment:
Cool - but note that doing `>&2` or whatever to lose stderr would be wrong. We only suppress stdout because it generates a huge amount of pointless output (one line per unzip'd file). stderr is still super valuable.